internal/az/registry.go

```diff
@@ -55,37 +55,101 @@ func RegisterAzCommand(cmd AksCommand) mcp.Tool {
 		{Name: "az aks nodepool upgrade", Description: "Upgrade a node pool to a newer version", ArgsExample: "--cluster-name myAKSCluster --resource-group myResourceGroup --name nodepool1 --kubernetes-version 1.28.0"},
+		{Name: "az aks nodepool update", Description: "Update a node pool properties", ArgsExample: "--cluster-name myAKSCluster --resource-group myResourceGroup --name nodepool1 --enable-cluster-autoscaler"},
+		{Name: "az aks nodepool start", Description: "Start stopped agent pool in the managed Kubernetes cluster", ArgsExample: "--cluster-name myAKSCluster --resource-group myResourceGroup --name nodepool1"},
+		{Name: "az aks nodepool stop", Description: "Stop running agent pool in the managed Kubernetes cluster", ArgsExample: "--cluster-name myAKSCluster --resource-group myResourceGroup --name nodepool1"},
+		{Name: "az aks nodepool operation-abort", Description: "Abort last running operation on nodepool", ArgsExample: "--cluster-name myAKSCluster --resource-group myResourceGroup --name nodepool1"},
+		{Name: "az aks nodepool delete-machines", Description: "Delete specific machines in an agentpool for a managed cluster", ArgsExample: "--cluster-name myAKSCluster --resource-group myResourceGroup --name nodepool1 --machine-names machine1"},
+
+		// Addon management
+		{Name: "az aks enable-addons", Description: "Enable Kubernetes addons", ArgsExample: "--addons monitoring --name myAKSCluster --resource-group myResourceGroup"},
+		{Name: "az aks disable-addons", Description: "Disable Kubernetes addons", ArgsExample: "--addons monitoring --name myAKSCluster --resource-group myResourceGroup"},
+		{Name: "az aks approuting enable", Description: "Enable App Routing addon for a managed Kubernetes cluster", ArgsExample: "--name myAKSCluster --resource-group myResourceGroup"},
+		{Name: "az aks approuting disable", Description: "Disable App Routing addon for a managed Kubernetes cluster", ArgsExample: "--name myAKSCluster --resource-group myResourceGroup"},
+
+		// Snapshot commands
+		{Name: "az aks snapshot create", Description: "Create a snapshot of a cluster", ArgsExample: "-g MyResourceGroup -n snapshot1 --cluster-id \"/subscriptions/00000/resourceGroups/AnotherResourceGroup/providers/Microsoft.ContainerService/managedClusters/akscluster1\""},
+		{Name: "az aks snapshot delete", Description: "Delete a cluster snapshot", ArgsExample: "--name myAKSSnapshot --resource-group myResourceGroup"},
+		{Name: "az aks nodepool snapshot create", Description: "Create a nodepool snapshot", ArgsExample: "-g MyResourceGroup -n snapshot1 --nodepool-id \"/subscriptions/00000/resourceGroups/AnotherResourceGroup/providers/Microsoft.ContainerService/managedClusters/akscluster1/agentPools/nodepool1\""},
+		{Name: "az aks nodepool snapshot delete", Description: "Delete a nodepool snapshot", ArgsExample: "--name myNodepoolSnapshot --resource-group myResourceGroup"},
+
+		// Maintenance commands
```

Comment:
number of tools are too big. Maybe we can add a flag to enable non-essential tools, e.g. --enable-all-tools?

Reply:
talked offline, we will only add the tools that on the list of one-pager for now. Comment out tools that are not yet agreed and add a description in Line 55 to show why they are commented out.